js/client-teambuilder.js

```diff
@@ -355,13 +355,19 @@
 				}
 			}
 
-			var newButtonText = "New Team";
-			if (filterFolder) newButtonText = "New Team in folder";
+			var newTeamButtonText = "New Team";
+			var newBoxButtonText = "New Box";
+			if (filterFolder) {
+				newTeamButtonText = "New Team in folder";
+				newBoxButtonText = "New Box in folder";
+			}
 			if (filterFormat && filterFormat !== 'gen8') {
-				newButtonText = "New " + BattleLog.escapeFormat(filterFormat) + " Team";
+				newTeamButtonText = "New " + BattleLog.escapeFormat(filterFormat) + " Team";
+				newBoxButtonText = "New " + BattleLog.escapeFormat(filterFormat) + " Box";
 			}
-			buf += '<p><button name="newTop" class="button big"><i class="fa fa-plus-circle"></i> ' + newButtonText + '</button> ' +
-					 '<input type="text" id="teamSearchBar" name="search" class="textbox searchinput" value="' + this.curSearchVal + '" placeholder="search teams"/></p>';
+			buf += '<p><button name="newTop" value="team" class="button big"><i class="fa fa-plus-circle"></i> ' + newTeamButtonText + '</button> ' +
+				'<button name="newTop" value="box" class="button big"><i class="fa fa-plus-circle"></i> ' + newBoxButtonText + '</button> ' +
```

Comment:
This one could stand to use another icon.

Reply:
Square plus is the boring way:

https://fontawesome.com/v4.7.0/icon/plus-square

Thinking like a shopping cart:

https://fontawesome.com/v4.7.0/icon/shopping-bag

https://fontawesome.com/v4.7.0/icon/shopping-basket

https://fontawesome.com/v4.7.0/icon/shopping-cart

Referencing "the PC" in pokemon games:

https://fontawesome.com/v4.7.0/icon/tablet

https://fontawesome.com/v4.7.0/icon/television

https://fontawesome.com/v4.7.0/icon/laptop

https://fontawesome.com/v4.7.0/icon/server

Abstract:

https://fontawesome.com/v4.7.0/icon/sitemap

https://fontawesome.com/v4.7.0/icon/object-group

https://fontawesome.com/v4.7.0/icon/window-restore

https://fontawesome.com/v4.7.0/icon/upload

https://fontawesome.com/v4.7.0/icon/tags

https://fontawesome.com/v4.7.0/icon/th

https://fontawesome.com/v4.7.0/icon/list-alt

Reply:
![image](https://user-images.githubusercontent.com/551184/98137438-d7c63f00-1e76-11eb-964e-8185a226534f.png)

Front-runner imo

Reply:
What about this one? https://fontawesome.com/v4.7.0/icon/archive

Reply:
Oh I like that one!